src/libraries/System.Drawing.Common/tests/Drawing2D/LinearGradientBrushTests.cs

```diff
@@ -358,6 +358,17 @@ public void Blend_SetNullBlendFactors_ThrowsNullReferenceException()
         }
 
         [ConditionalFact(Helpers.IsDrawingSupported)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Exception not updated on NetFX")]
+        public void Blend_SetNullBlendPositions_ThrowsArgumentException()
+        {
+            using (var brush = new LinearGradientBrush(new Rectangle(1, 2, 3, 4), Color.Plum, Color.Red, 45, true))
+            {
+                AssertExtensions.Throws<ArgumentException>("value", () => brush.Blend = new Blend { Factors = new float[2], Positions = null });
+            }
+        }
+
+        [ConditionalFact(Helpers.IsDrawingSupported)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp, "Only for NetFX")]
```

Comment:
Not sure if we want to test version for .Net framework, added just in case, please let me know if we don't need this